src/Vendr.Checkout/PathHelper.cs

```diff
@@ -1,14 +1,22 @@
 ﻿using System;
+using System.Configuration;
 
 namespace Vendr.Checkout
 {
     public static class PathHelper
     {
-        public const string RootPath = "/App_Plugins/VendrCheckout";
-        public const string VirtualRootPath = "~" + RootPath;
-        private const string VirtualViewPathToken = "~" + RootPath + "/Views/{0}.cshtml";
-        private const string VirtualPartialViewPathToken = "~" + RootPath + "/Views/Partials/{0}.cshtml";
-        private const string VirtualEmailViewPathToken = "~" + RootPath + "/Views/Emails/{0}.cshtml";
+        public static string RootPath
+        {
+            get
+            {
+                var rootPath = ConfigurationManager.AppSettings["vendr:RootPath"]?.ToString();
```

Comment:
I'd probably set the app setting prefix to be `VendrCheckout:` rather than `Vendr` just so it's not confused as being a core product setting

Reply:
Pefect!